src/django/api/csv_download.py

```diff
@@ -0,0 +1,97 @@
+from api.models import ExtendedField
+
+from api.helpers import prefix_a_an
+
+
+def format_download_date(date):
+    return date.strftime("%d-%b-%y")
```

Comment:
It was formatted that way in the demo download from the OAR team, but I think it does make more sense to use `YYYY-MM-DD` since we use that elsewhere. Updated.